### ISSUE TITLE
New crosshair cvars, crosshair_drawoutline + crosshair_outlinethickness

### ISSUE THREAD

**Issue by samersultan** [Skarbo/CSGOCrosshair#1]
Would it be possible to add the new crosshair outline commands to the UI. 

They are: 

cl_crosshair_drawoutline "x" (0 for disabled, 1 for enabled)
cl_crosshair_outlinethickness "x" (0.1 - 1)

**Comment by Skarbo** [comment]
Yes, that is in the plans. I'm just finishing up school. Give me 2 weeks. Thanks for using the site!

**Comment by samersultan** [comment]
Sure thing, let me know if you need my help on this project.

**Comment by megaroeny** [comment]
Yeah just realized that the outline commands are wrong. You still going to fix this? :/

**Comment by ottk3** [comment]
Would be great to implement the new commands.
